yapapi/executor/__init__.py

```diff
@@ -751,7 +757,7 @@ def _get_allocation(
                 and allocation.payment_platform == item.payment_platform
             )
         except:
-            raise RuntimeError(f"No allocation for {item.payment_platform} {item.payer_addr}.")
+            raise ValueError(f"No allocation for {item.payment_platform} {item.payer_addr}.")
```

Comment:
This is not related to other changes in this PR. I've just thought that since we cannot guarantee that the `item` argument is valid (since it may come from a response to an API call), `ValueError` would be more appropriate.